components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDLSOAPOperationExtractor.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
```

Comment:
Year should be 2018

Reply:
fixed in e18f43f5dee9ed43a3239af62f38d67c72c78fa2